stdlib/public/Distributed/DistributedActorSystem.swift

```diff
@@ -569,10 +573,28 @@ public protocol DistributedActorSystemError: Error {}
 
 @available(SwiftStdlib 5.7, *)
 public struct ExecuteDistributedTargetError: DistributedActorSystemError {
-  let message: String
+  public let errorCode: ErrorCode
+  public let message: String
+
+  public enum ErrorCode {
+    /// Thrown when unable to resolve the target identifier to a function accessor.
+    /// This can happen when the identifier is corrupt, illegal, or wrong in the
+    /// sense that the caller and callee do not have the called function recorded
+    /// using the same identifier.
+    case targetAccessorNotFound
+
```

Comment:
Could you please add types for all other errors we currently have?

- decoding failures:
  - invalid parameter count
  - missing generic substitutions
  - invalid generic substitutions
  - type deserialization failure (parameters/result)

Reply:
Okey, let's do that right away here 👍

Reply:
done in https://github.com/apple/swift/pull/60476/commits/82e916a6d2908fed28c30088e43d1f649dd8700f also on main in https://github.com/apple/swift/pull/60480